specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json

```diff
@@ -253,6 +253,57 @@
           "description": "Whether the operation can be cancelled.",
           "type": "boolean",
           "readOnly": true
+        },
+        "operationParameters": {
+          "$ref": "#/definitions/ManagedInstanceOperationParametersPair",
+          "description": "The operation parameters.",
+          "readOnly": true
+        },
+        "operationSteps": {
+          "$ref": "#/definitions/ManagedInstanceOperationSteps",
+          "description": "The operation steps.",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedInstanceOperationParametersPair": {
```

Comment:
The name of the definition will appear as a type/class name in the generated SDKs. Are these guaranteed to be pairs forever and ever?

Reply:
If not, you should rename the definition.

Reply:
Yes.